src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -49,6 +49,19 @@ public function __construct($controller, LoggerInterface $logger = null, $debug
 
     public function onKernelException(GetResponseForExceptionEvent $event)
     {
+        if (null === $this->controller) {
+            $terminating = false;
+            foreach (debug_backtrace() as $frame) {
```

Comment:
I'm very much against this kind of code.

Reply:
I agree, I lacked a better idea, but this is fixed now.